app/frontend/src/features/communities/CommunityPage/HeaderImage.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, makeStyles } from "@material-ui/core";
+import { LngLat } from "maplibre-gl";
+import React from "react";
+
+import Map from "../../../components/Map";
+import { Community } from "../../../pb/communities_pb";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100vw",
+    height: "8rem",
+    [theme.breakpoints.up("md")]: {
+      position: "relative",
+      left: "50%",
+      right: "50%",
+      marginLeft: "-50vw",
+      marginRight: "-50vw",
```

Comment:
It's to escape the container on mobile (changed to `breakpoints.down["sm"]`, and display full width. I prefer this solution than using the newly-implemented full screen routes, since I want the container except for this component.

> Can we be sure 50% is always 50vw though?
No, but it should be if it's used in a centered container. [Taken from here.
](https://css-tricks.com/full-width-containers-limited-width-parents/#no-calc-needed)

Reply:
Amazing... the funky tricks available in CSS is mind bending sometimes 😮